deepfence_console/elastic-Dockerfile

```diff
@@ -83,11 +82,13 @@ USER root
 #COPY --from=build /go/check-postgres /usr/local/bin/check-postgres
 RUN mkdir /data \
     && chmod 755 /usr/bin/startEs.sh \
-    && chown -R elasticsearch:elasticsearch /data \
-    && apk add --no-cache zip \
-    && zip -q -d /usr/share/elasticsearch/lib/log4j-core-2.11.1.jar org/apache/logging/log4j/core/lookup/JndiLookup.class \
-    && apk del --purge zip
-#&& chmod 755 /usr/local/bin/check-postgres
+    && chown -R elasticsearch:elasticsearch /data
+#    && apk add --no-cache zip \
+#    && apk del --purge zip \
+#    && chmod 755 /usr/local/bin/check-postgres \   
+#    && ls org/apache/logging/log4j/core/lookup/JndiLookup.class \
+#    && zip -q -d /usr/share/elasticsearch/lib/log4j-core-2.11.1.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
```

Comment:
if this PR also upgrades the log4j version there is no need to remove `JndiLookup.class`.

@ramanan-ravi pls. confirm

Reply:
These are commented out right now. We can remove them.